Applications/Tizen_native/CustomShortcut/src/main.c

```diff
@@ -0,0 +1,168 @@
+/**
+ * @file main.c
+ * @date 14 May 2020
+ * @brief TIZEN Native Example App main entry with NNTrainer/CAPI.
+ * @see  https://github.com/nnstreamer/nntrainer
+ * @author Jihoon Lee <jhoon.it.lee@samsung.com>
+ * @bug No known bugs except for NYI items
+ */
+
+#include <main.h>
+#include <tizen.h>
+
+typedef struct widget_instance_data {
+	Evas_Object *win;
+	Evas_Object *conform;
+	Evas_Object *label;
+} widget_instance_data_s;
+
+static int
+widget_instance_create(widget_context_h context, bundle *content, int w, int h, void *user_data)
+{
+	widget_instance_data_s *wid = (widget_instance_data_s*) malloc(sizeof(widget_instance_data_s));
```

Comment:
All Tizen native apps as well as nnstreamer use 2 spaces indentation instead of Tab.

Reply:
Fixed. Thanks 😃